charts/redpanda/testdata/30-additional-flags-override-values.yaml.golden

```diff
@@ -0,0 +1,1402 @@
+---
```

Comment:
This file is too old. Please remote `charts/redpanda/Chart.lock` and re-run.

Reply:
https://github.com/redpanda-data/helm-charts/actions/runs/8559264006/job/23455747231?pr=1151#step:5:293